src/Compilers/CSharp/Portable/Symbols/ReducedExtensionMethodSymbol.cs

```diff
@@ -621,6 +621,27 @@ public override ImmutableArray<CustomModifier> RefCustomModifiers
                 }
             }
 
+            internal override bool IsCallerLineNumber
+            {
+                get { return _underlyingParameter.IsCallerLineNumber; }
+            }
+
+            internal override bool IsCallerFilePath
+            {
+                get { return _underlyingParameter.IsCallerFilePath; }
+            }
+
+            internal override bool IsCallerMemberName
+            {
+                get { return _underlyingParameter.IsCallerMemberName; }
+            }
+
+            internal override int CallerArgumentExpressionParameterIndex
+            {
+                // ReducedExtensionMethodParameterSymbol is only exposed to semantic model.
+                get { throw ExceptionUtilities.Unreachable; }
```

Comment:
>ExceptionUtilities.Unreachable; [](http://example.com/codeflow?start=28&length=31)

Do we have tests covering CallerArgumentExpression for extension methods in C#?

Reply:
@AlekseyTs Yes there are.